tests/pipelines_components/automl/upgrade/conftest.py

```
@@ -0,0 +1,976 @@
+)
+from utilities.general import collect_pod_information
+from utilities.inference_utils import create_isvc
+from utilities.infra import create_ns, wait_for_dsc_status_ready
+
+LOGGER = structlog.get_logger(name=__name__)
+
+UPGRADE_NAMESPACE = "automl-upgrade"
+UPGRADE_TASK_TYPE = "regression"
+UPGRADE_RUN_DISPLAY_NAME = "automl-upgrade-regression"
+UPGRADE_ISVC_NAME = "automl-upgrade-model"
+KSERVE_S3_SECRET_NAME = "kserve-minio-credentials"  # pragma: allowlist secret
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "KSERVE_S3_SECRET_NAME"

(S105)